audio/table.go

```diff
@@ -0,0 +1,78 @@
+package audio
```

Comment:
Yup, but set the year to 2015.

On Fri, Feb 13, 2015 at 11:51 AM, Kamil Kisiel <notifications@github.com>
wrote:

> In audio/table.go
> <https://github.com/nf/sigourney/pull/10#discussion_r24636169>:
>
> > @@ -0,0 +1,78 @@
> > +package audio
>
> Should I just use the notice from the other files?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/nf/sigourney/pull/10/files#r24636169>.
>